velox/exec/fuzzer/AggregationFuzzer.cpp

```diff
@@ -111,6 +113,15 @@ class AggregationFuzzer : public AggregationFuzzerBase {
   void updateReferenceQueryStats(
       AggregationFuzzerBase::ReferenceQueryErrorCode errorCode);
 
+  bool compareEquivalentPlanResults(
+      const core::PlanNodePtr& plan,
```

Comment:
`plan` can be obtained from `plans[0]`, so no need to pass it additionally. See line 841 (original 725).